cumulusci/core/config/org_config.py

```diff
@@ -204,7 +205,15 @@ def user_id(self):
 
     @property
     def org_id(self):
-        return self.id.split("/")[-2]
+        try:
```

Comment:
This code probably relied previously on implicit None for config objects which is evil.